frontend/src/components/SensorPreview.vue

```diff
@@ -1,39 +1,45 @@
 <template>
   <v-card v-ripple class="parent">
-    <v-row no-gutters><v-icon class="icon">{{ icon }}</v-icon></v-row>
-    <v-row no-gutters><div class="name">{{ name }}</div></v-row>
-    <v-row no-gutters><h1>{{ data }}</h1></v-row>
+    <v-row no-gutters
+      ><v-icon class="icon">{{ icon }}</v-icon></v-row
+    >
```

Comment:
Change the `v-row` tags so they aren't split up onto separate lines.

```suggestion
    <v-row no-gutters>
        <v-icon class="icon">{{ icon }}</v-icon>
    </v-row>
```